src/governance-classes.cpp

```diff
@@ -512,9 +509,31 @@ CSuperblock(uint256& nHash)
     DBG( cout << "CSuperblock Constructor End" << endl; );
 }
 
-void 
-CSuperblock::
-ParsePaymentSchedule(std::string& strPaymentAddresses, std::string& strPaymentAmounts)
+bool CSuperblock::IsValidBlockHeight(int nBlockHeight)
```

Comment:
I don't really see the rationale for moving this from CSuperblockManager to CSuperblock.

Either way I believe this function should be inlined for efficiency.  It gets called a lot (ie. for every new block) and even with your changes contains little code.